### ISSUE TITLE
Show error message when no connection can be established

### ISSUE THREAD

**Issue by scheidan** [karthink/gptel#26]
Thanks for the great package!

It would be very helpful if we could see the error messages in cases the connection did not work. For example when using curl we get a helpful hint:
```
 curl https://api.openai.com/v1/chat/completions \                                                    (base)
                                     -H "Content-Type: application/json" \
                                     -H "Authorization: Bearer sk-K6F..." \
                                     -d '{
                                 "model": "gpt-3.5-turbo",
                                 "messages": [{"role": "user", "content": "Say this is a test!"}],
                                 "temperature": 0.7
                                 }'
```
leads to:
```
{
    "error": {
        "message": "You exceeded your current quota, please check your plan and billing details.",
        "type": "insufficient_quota",
        "param": null,
        "code": null
    }
}
```

Currently in Emacs it only shows `*ChatGPT* Response Error: finished`.

**Comment by karthink** [comment]
I just addressed this in #10 a couple of days ago. Just checking, are you on the latest commit?

**Comment by scheidan** [comment]
Sorry for the noise! I made a mess with updating. Works fine now :)
